src/parser.rs

```diff
@@ -136,11 +137,18 @@ impl<'a> DocParser<'a> {
         ))
       })?;
 
-    self.parse_source(
-      &module.specifier,
-      module.media_type,
-      module.source.clone(),
-    )
+    if let Module::Es(module) = module {
+      self.parse_source(
+        &module.specifier,
+        module.media_type,
+        module.source.clone(),
+      )
+    } else {
+      Err(DocError::Resolve(format!(
+        "{} is not an JavaScript/TypeScript module",
+        specifier
+      )))
```

Comment:
Will this be a fatal error for https://doc.deno.land? If there's appropriate prompt then it's okay

Reply:
No, it is likely to continue to break at the build of the graph.  If you try to doc a JSON module today you get:

![https___doc_deno_land_https___deno_land_std_0_117_0_deno_json](https://user-images.githubusercontent.com/1282577/145928090-f31706b5-74e9-485f-a394-eb807e3cb8e9.png)